types/office-js/index.d.ts

```diff
@@ -26267,12 +26267,12 @@ declare namespace Excel {
     /**
      * Notifies when a worksheet is moved within a workbook.
                 
-                If a worksheet is moved from one position within the workbook to another
-                via the Excel UI, then this API will trigger an event. Note that if
-                the position of a worksheet changes as a result of moving a different worksheet,
-                then this event won't trigger for both position changes. This event only triggers
-                for the primary worksheet move, and not any worksheet position changes that occur
-                as a result of that primary move.
+                 If a worksheet is moved from one position within the workbook to another
```

Comment:
@vaquierm Both the prod and beta versions of office.d.ts on the CDN include new single spaces at the start of multiple lines, like this one. There are approx 30 groups of lines with these new single spaces, so maybe 80 lines total. 

I removed the new spaces from the beta version of office.d.ts (office-js-preview.index.d.ts) in this PR, but I left them in the prod version (office-js/index.d.ts) so you can take a look. Any idea why this happened? Thanks!

Reply:
Looking into it.